test/misc/unit/libraries/TransientBytesLib.t.sol

```diff
@@ -17,4 +17,14 @@ contract TransientBytesLibTest is Test {
 
         assertEq(TransientBytesLib.get(invalidKey).length, 0);
     }
+
+    function testAppend(bytes calldata data1, bytes calldata data2) public {
+        bytes32 key = keccak256(abi.encode("key"));
+
+        TransientBytesLib.set(key, data1);
+        assertEq(TransientBytesLib.get(key), data1);
+
+        TransientBytesLib.append(key, data2);
```

Comment:
Can we add a second `append` in here with `data3`, to check appending multiple times works well?

Reply:
@lemunozm could you still add this in a followup PR?

Reply:
My bad I misread!